test/upgrade/mzcompose.py

```diff
@@ -77,7 +77,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         "--min-version",
         metavar="VERSION",
         type=MzVersion.parse,
-        default=MzVersion.parse("0.39.0"),
+        default=MzVersion.parse("0.52.0"),
```

Comment:
Make sure to get someone from QA to look at this.

Reply:
cc @def- what do you think about this?

Reply:
All customers being on 0.52.0 already makes this seem reasonable to me.
I haven't actually touched these legacy upgrade tests yet, but based on what I can see this should still work fine, and since chain upgrades were not tested, shouldn't be affected.

If this is not urgent you could wait until tomorrow to see if @philip-stoev has a different opinion, he's probably the expert on this.

Reply:
This particular option here is/was not used since CI runs this test with `--most-recent=2`, which overrides the min-version.